10up/brightcove-video-connect
# Fix label fields on video editing

### Description of the Change

The field label is not sent on the form submit when a video is updated. This PR fixes that.

### Alternate Designs



### Benefits

The user can add/remove labels to a video.

### Possible Drawbacks



### Verification Process

1. Edit a video and add/remove labels
2. Save the video
2. Refresh the page and check if the value for the label is correct

### Checklist:






### Applicable Issues



### Changelog Entry

Fixed label fields on video editing.
